README.md

```diff
@@ -89,25 +89,9 @@ Milvus is designed to handle vector search at scale. Users can store vectors, wh
 
 Milvus is trusted by AI developers to build applications such as text and image search, Retrieval-Augmented Generation (RAG), and recommendation systems. Milvus powers [many mission-critical business]((https://milvus.io/use-cases)) for startups and enterprises.
 
-## Demos and Tutorials 
-Here is a selection of demos and tutorials to show how to build various types of AI applications made with Milvus:
+## Demos and Tutorials
 
-| Tutorial | Use Case | Related Milvus Features | 
-| -------- | -------- | --------- |
-| [Build RAG with Milvus](build-rag-with-milvus.md) |  RAG | vector search |
-| [Multimodal RAG with Milvus](multimodal_rag_with_milvus.md) | RAG | vector search, dynamic field |
-| [Image Search with Milvus](image_similarity_search.md) | Semantic Search | vector search, dynamic field |
-| [Hybrid Search with Milvus](hybrid_search_with_milvus.md) | Hybrid Search | hybrid search, multi vector, dense embedding, sparse embedding |
-| [Multimodal Search using Multi Vectors](multimodal_rag_with_milvus.md) | Semantic Search | multi vector, hybrid search |
-| [Recommender System](recommendation_system.md) | Recommendation System | vector search |
-| [Video Similarity Search](video_similarity_search.md) | Semantic Search | vector search |
-| [Audio Similarity Search](audio_similarity_search.md) | Semantic Search | vector search |
-| [DNA Classification](dna_sequence_classification.md) | Classification | vector search |
-| [Graph RAG with Milvus](graph_rag_with_milvus.md) | RAG | vector search |
-| [Contextual Retrieval with Milvus](contextual_retrieval_with_milvus.md) | RAG, Semantic Search | vector search |
-| [HDBSCAN Clustering with Milvus](hdbscan_clustering_with_milvus.md) | Clustering | vector search |
-| [Use ColPali for Multi-Modal Retrieval with Milvus](use_ColPali_with_milvus.md) | RAG, Semantic Search | vector search |
-| [Vector Visualization](vector_visualization.md) | Data Visualization | vector search |
+Here is a selection of demos and tutorials to show how to build various types of AI applications made with Milvus:
```

Comment:
shall we not just remove the list of demos and tutorials, but correct the link to each demo and tutorial?

Reply:
I updated the table and tutorial links based on [this link](https://raw.githubusercontent.com/milvus-io/milvus-docs/refs/heads/v2.5.x/site/en/tutorials/tutorials-overview.md), and added the table for the Chinese documentation.(all links available)